app/services/update_user.rb

```diff
@@ -33,7 +33,24 @@ def update_sam
     reckoner.delay.set!
   end
 
+  def parsed_user_params
+    user_params.each_with_object({ }) do |(key, value), hash|
+      converted_value = nil_not_empty_string(value)
+      hash[key] = converted_value
+    end
+  end
```

Comment:
Why not process an empty string DUNS number in the validator? https://github.com/18F/micropurchase/blob/develop/app/validators/duns_number_validator.rb

That's where I would think this logic belongs. It feels inefficient to me to call `nil_not_empty_string(value)` for every param when the only one you care about is the DUNS number.